include/z64animation.h

```diff
@@ -204,26 +204,26 @@ typedef struct SkelAnime {
 } SkelAnime; // size = 0x44
 
 typedef s32 (*OverrideLimbDrawOpa)(struct GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
-                                   struct Actor* actor);
+                                   struct Actor* thisx);
```

Comment:
This has been limited to `Actor*` because we checked every use of the override/post limb draw callbacks and all of them pass an `Actor*`, so it was decided to use that instead of a `void*` like OoT does

Reply:
And we were concerned about having room on the stack for recasts, I think?